src/cmd/package_test.go

```diff
@@ -189,6 +196,10 @@ func TestPackageInspectManifests(t *testing.T) {
 			}
 			packagePath := filepath.Join(tmpdir, fmt.Sprintf("zarf-package-%s-%s.tar.zst", tc.packageName, config.GetArch()))
 			err = opts.run(context.Background(), []string{packagePath})
+			if tc.expectedErr != "" {
+				require.EqualError(t, err, tc.expectedErr)
```

Comment:
nit: `require.ErrorContains` or alternative, we want to avoid tests that act as change detectors. https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html